R/emmeans.rma.R

```diff
@@ -115,96 +115,3 @@ addSignifStars <- function(x, ..., column = "P(>|t|)", stars = c(' ***'=.001, '
   names(X) = ifelse(is.na(names(X)), ' ', names(X))
   X
 }
-
-(function(conf.level = .95){
-  formula = ~  colour
-  data = carData::Arrests
-  fit = lm(checks ~ colour*sex+employed, data = data)
-  mf = if(is.data.frame(model.frame(fit))) model.frame(fit) else data[, all.vars(mdlformula)]
-  
-  mdlformula = fit$call$formula
-  mdlformula[[2]] = NULL # remove response variable
-  
-  b = coef(fit)
-  Vb = vcov(fit)
-  
-  tmp = expand.grid(Filter(Negate(is.null), lapply(mf, levels)))
-  #browser()
-  M = do.call(model.matrix, list(mdlformula, tmp)) #, contrasts = list(wool='contr.Treatment', tension='contr.Treatment')))
-  Kmf = model.frame(paste("~ 0 + ",all.vars(formula), collapse = ":"), unique(tmp[, all.vars(formula), drop = FALSE]))
-  K = model.matrix(Kmf, tmp, contrasts = as.list(setNames(rep("contr.Treatment", length(all.vars(formula))), all.vars(formula))))
-  K.M = ginv(K) %*% M
-  em = K.M %*% b
-  se = diag(K.M %*% Vb %*% t(K.M))^.5
-  ret = transform(Kmf, emmean = em, SE = se, df = df.residual(fit))
-  structure(transform(ret, lower.CL = emmean - qt(.025, df)*se, upper.CL = emmean - qt(0.5-conf.level/2, df)*se), 
-            collapsed = setdiff(all.vars(mdlformula), all.vars(formula)), conf.level = conf.level)
-})#() # () # for testing the correctness
```

Comment:
Any reason in particular we had this @FBartos?

Reply:
no idea, this part was probably implemented by @sophieberkhout / Raul?

Reply:
I don't know what this is.

Reply:
Ah, yes, this seems to have been around for a bit.